### ISSUE TITLE
python: Can't find mrpt/utils/CRobotSimulator.h

### ISSUE THREAD

**Issue by jolting** [MRPT/mrpt#278]
I'm getting a build error in the python bindings.
https://launchpadlibrarian.net/267300479/buildlog_ubuntu-xenial-amd64.mrpt_1%3A1.5.0~snapshot201606202240-git-e9dfae2-201606231934~ubuntu16.04.1_BUILDING.txt.gz

```
/«PKGBUILDDIR»/python/src/utils_bindings.cpp:14:40: fatal error: mrpt/utils/CRobotSimulator.h: No such file or directory
```

**Comment by jlblancoc** [comment]
Solved after #280

**Comment by jolting** [comment]
@jlblancoc
Everything appears to be building as of 5151dcb

https://launchpad.net/~jolting/+archive/ubuntu/mrpt-daily/+build/10178464
